### DISCUSSION TITLE
Can we have access to mockups (PSD or other) files ?

### DISCUSSION THREAD

**Discussion by francedeveloppement** [ColorlibHQ/AdminLTE#4258] [Q&A] [answered]
Hello guys,

First, I wanted to say that AdminLTE makes us gain lot of times! We found everything we need to develop our own application.

I wonder if we have an access to mockup ? Whatever kind of files, this could be awesome to have PSD, AI files.
I'm project manager and it could be easier to create mockup for our dev team.

Have a good day !
Maxime

**Comment by lenamtl** [comment, marked as answer]
Hi,

Nowadays we rarely work with artwork files for web page because everything is mostly done using CSS this allow to be responsive 
(adapt to device width).

What you can do to is to take print screen of elements from the demo .
Or simply use a mockup tool with some reference to demo page or printscreen.

*Note depending of the device width the elements and the place where they appear on the page can be different.

**Reply by REJack** [reply]
There is no real mockup or something similar, like a friend of me says "its naturally grown" 🤣 I never got something like this to keep the style so we decide with every new version what design change we make
